lib/components/util/sortOptions.ts

```diff
@@ -1,26 +1,44 @@
-export const sortOptions = [
-  {
-    locale: 'components.NarrativeItinerariesHeader.selectBest',
-    value: 'BEST'
-  },
-  {
-    locale: 'components.NarrativeItinerariesHeader.selectDuration',
-    value: 'DURATION'
-  },
-  {
-    locale: 'components.NarrativeItinerariesHeader.selectArrivalTime',
-    value: 'ARRIVALTIME'
-  },
-  {
-    locale: 'components.NarrativeItinerariesHeader.selectDepartureTime',
-    value: 'DEPARTURETIME'
-  },
-  {
-    locale: 'components.NarrativeItinerariesHeader.selectWalkTime',
-    value: 'WALKTIME'
-  },
-  {
-    locale: 'components.NarrativeItinerariesHeader.selectCost',
-    value: 'COST'
+import { useIntl } from 'react-intl'
+
+export const sortOptions = (): {
```

Comment:
Better to pass `intl` to `sortOptions` so you can write attributes as below and get rid of `OptionText`:
```
text: intl.formatMessage('components.NarrativeItinerariesHeader...')
```